modules/transactions.js

```diff
@@ -721,26 +740,20 @@ Transactions.prototype.shared = {
 		);
 	},
 
+	postTransaction(transaction, cb) {
+		return modules.transport.shared.postTransaction(
+			{ transaction },
+			(err, res) => {
+				__private.processPostResult(err, res, cb);
```

Comment:
In reference to above comments, here it seems right that we are passing `err` and `res` both.